app/models/claim_student_loan_details_updater.rb

```diff
@@ -14,7 +14,7 @@ def update_claim_with_latest_data
       # When the claim hasn't been submitted yet, we need a way of knowing if the student loan
       # details on the claim were found using the SLC data we held before submission;
       # after submission, the `submitted_using_slc_data` value must not change
-      claim.assign_attributes(submitted_using_slc_data: found_data?) unless claim.submitted?
+      claim.assign_attributes(submitted_using_slc_data: student_loans_data.found_data?) unless claim.submitted?
```

Comment:
Question: Do we need to set `submitted_using_slc_data` on the journey session and write that to the claim in the claim submission form? If so probably shouldn't merge this pr yet!